datasets/medical_questions_pairs/README.md

```diff
@@ -0,0 +1,135 @@
+---
+{"task_categories": ["text-classification"], "task_ids": ["semantic-similarity-classification"], "multilinguality": ["monolingual"], "languages": ["en"], "language_creators": ["other"], "annotations_creators": ["expert-generated"], "source_datasets": ["original"], "size_categories": ["1K<n<10K"], "licenses": ["unknown"]}
```

Comment:
We expect the YAML tags to be used here, not JSON
Sorry it that wasn't clear

The YAML tags can be shown in the tagger in the upper right corner by clicking on `expand yaml tags`